internal/manager/controllerdef.go

```diff
@@ -197,7 +198,7 @@ func setupControllers(
 				IngressClassName:           c.IngressClassName,
 				DisableIngressClassLookups: !c.IngressClassNetV1Enabled,
 				StatusQueue:                kubernetesStatusQueue,
-				DataplaneAddressFinder:     dataplaneAddressFinder,
+				DataplaneAddressFinder:     udpDataplaneAddressFinder,
```

Comment:
I think `UDPRoute` controller should also use `udpDataplaneAddressFinder`.

Reply:
Nah, Gateway API-side it's tied to the Gateway, this only matters for our legacy CRD that mimics Ingress. No address info in route statuses: https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.RouteStatus